airbyte-integrations/connectors/source-gcs/source_gcs/source.py

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+import json
+from datetime import datetime
+from typing import Dict, Generator
+
+from airbyte_cdk.logger import AirbyteLogger
+from airbyte_cdk.models import (
+    AirbyteCatalog,
+    AirbyteConnectionStatus,
+    AirbyteMessage,
+    AirbyteRecordMessage,
+    AirbyteStream,
+    ConfiguredAirbyteCatalog,
+    Status,
+    Type,
+)
+from airbyte_cdk.sources import Source
+
+from .helpers import construct_file_schema, get_gcs_blobs, read_csv_file
+
+
+class SourceGCS(Source):
+    def check(self, logger: AirbyteLogger, config: json) -> AirbyteConnectionStatus:
+        """
+        Check to see if a client can be created and list the files in the bucket.
+        """
+        try:
+            blobs = get_gcs_blobs(config)
+            # TODO: only support CSV intially. Change this check if implementing other file formats.
+            files = [blob.name for blob in blobs if "csv" in blob.name.lower()]
+            if not files:
+                return AirbyteConnectionStatus(status=Status.FAILED, message=f"No compatible file found in bucket")
+            return AirbyteConnectionStatus(status=Status.SUCCEEDED)
+        except Exception as e:
+            return AirbyteConnectionStatus(status=Status.FAILED, message=f"An exception occurred: {str(e)}")
+
+    def discover(self, logger: AirbyteLogger, config: json) -> AirbyteCatalog:
+        streams = []
+
+        blobs = get_gcs_blobs(config)
+        for blob in blobs:
+            # Read the first 0.1MB of the file to determine schema
+            df = read_csv_file(blob, limit_bytes=102400)
```

Comment:
Can you create a meaningful variable for the value 102400? `MIN_BYTES_READ_GENERATE_SCHEMA`